test/test_charge_drift_models.jl

```diff
@@ -82,41 +82,109 @@ end
     @test all(signalsum_list .< T(2.0))
     @test all(diff(signalsum_list) .> 0)
 
-    config_dict = SolidStateDetectors.parse_config_file(SSD_examples[:TrueCoaxial])
-    @testset "Parse config file 1" begin
-        config_dict["detectors"][1]["semiconductor"]["charge_trapping_model"] = Dict(
-            "model" => "ConstantLifetime",
-            "parameters" => Dict(
-                "τh" => "1ms",
-                "τe" => "1ms",
-            )
-        )
-        simA = @test_nowarn Simulation{T}(config_dict)
-        @test simA.detector.semiconductor.charge_trapping_model isa ConstantLifetimeChargeTrappingModel{T}
-        @test simA.detector.semiconductor.charge_trapping_model.τh == T(0.001)
-        @test simA.detector.semiconductor.charge_trapping_model.τe == T(0.001)
+    # test3: model both the bulk/inactive volumes
+    ## test 3.1: the bulk/inactive signals while varying the lifetimes: τ, τ_inactive
+    pos_bulk = CartesianPoint{T}(0.0085,0,0.005); Edep = 1u"eV"
+    @test !in(pos_bulk, simA_inactive_layer_geometry)
+    signalsum_list_bulk = []
+    τ_list = (10 .^ (-2:0.2:0))*u"ms"
+    pos_inactive = CartesianPoint{T}(0.0095,0,0.005); Edep = 1u"eV"
+    @test in(pos_inactive, simA_inactive_layer_geometry)
+    signalsum_list_inactive = []
+    τ_inactive_list = τ_list/100
+    for (τ,τ_inactive) in zip(τ_list, τ_inactive_list)
+        parameters = Dict("parameters" => Dict("τh" => τ, "τe" => τ, "τh_inactive" => τ_inactive, "τe_inactive" => τ_inactive),
+            "inactive_layer_geometry" => simA_inactive_layer_geometry)
+        trapping_model=ConstantLifetimeChargeTrappingModel{T}(parameters)
+        simA.detector = SolidStateDetector(simA.detector, trapping_model)
+        evt_bulk = Event(pos_bulk , Edep)
+        timed_simulate!(evt_bulk, simA)
+        signalsum_bulk  = T(0)
+        for i in 1:length(evt_bulk.waveforms)
+            signalsum_bulk  += abs(ustrip(evt_bulk.waveforms[i].signal[end]))
+        end
+        signalsum_bulk  *= inv(ustrip(SolidStateDetectors._convert_internal_energy_to_external_charge(simA.detector.semiconductor.material)))
+        push!(signalsum_list_bulk, signalsum_bulk)
+
+        evt_inactive = Event(pos_inactive , Edep)
+        timed_simulate!(evt_inactive, simA)
+        signalsum_inactive  = T(0)
+        for i in 1:length(evt_inactive.waveforms)
+            signalsum_inactive  += abs(ustrip(evt_inactive.waveforms[i].signal[end]))
+        end
+        signalsum_inactive  *= inv(ustrip(SolidStateDetectors._convert_internal_energy_to_external_charge(simA.detector.semiconductor.material)))
+        push!(signalsum_list_inactive, signalsum_inactive)
     end
-
+    @info signalsum_list_bulk
+    @test all(signalsum_list_bulk .< T(2.0))
+    @test all(diff(signalsum_list_bulk) .> 0)
+    @info signalsum_list_inactive
+    @test all(signalsum_list_inactive .< T(2.0))
+    @test all(diff(signalsum_list_inactive) .> 0)
+    @test all(signalsum_list_bulk .> signalsum_list_inactive)
```

Comment:
Maybe remove the `@info` statements for the tests
```suggestion
    @test all(signalsum_list_bulk .< T(2.0))
    @test all(diff(signalsum_list_bulk) .> 0)
    @test all(signalsum_list_inactive .< T(2.0))
    @test all(diff(signalsum_list_inactive) .> 0)
    @test all(signalsum_list_bulk .> signalsum_list_inactive)
```